.github/workflows/python-package-conda.yml

```diff
@@ -0,0 +1,52 @@
+name: Conda-based pytest
+
+on: [push, pull_request]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: [2.7, 3.6, 3.9]
+        slycot: ["", "conda"]
+        array-and-matrix: [0]
+        include:
+          - python-version: 3.9
+            slycot: conda
+            array-and-matrix: 1
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
```

Comment:
Conflicts with the conda setup

Reply:
I think I might be able to remove this completely.  I meant to do that at one point but lost track of it while playing around to just get something to work.  I'll see if I can remove it and if things still work correctly.